python/ql/test/library-tests/PointsTo/calls/Argument.ql

```diff
@@ -1,5 +1,5 @@
 import python
 
-from ControlFlowNode arg, FunctionObject func, int i
+from ControlFlowNode arg, FunctionValue func, int i
```

Comment:
Not that this should be changed, but might this be generalized to `CallableValue` instead of just `FunctionValue`?

Reply:
I didn't care too much about the argument test, so only did a minimal change.